.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 12 * * 0'  # run once a week on Sunday
```

Comment:
Because of https://github.com/ecordell/pymacaroons/pull/60 - my PR was green a year ago when I created it, but there will be changes which break CI, e.g. updated dependencies, transitive dependencies or updated system packages / os. It is just a best practice to run CI jobs weekly to fix such breakages in a timely manner, and not e.g. when one needs to fix an urgent security issue.